deploy/helm/values/uat.yaml

```diff
@@ -49,4 +49,4 @@ threshold:
   useTestData: 'true'
 
 legalFrameworkApi:
-  host: https://legal-framework-api-staging.cloud-platform.service.justice.gov.uk
+  host: http://legal-framework-api.legal-framework-api-uat.svc.cluster.local
```

Comment:
This should be staging - we seem to follow a pattern of being 1 promotion ahead of our downstream systems (note previous URL was staging)